app/serializers/activitypub/note_serializer.rb

```diff
@@ -55,7 +55,7 @@ def replies
         type: :unordered,
         part_of: ActivityPub::TagManager.instance.replies_uri_for(object),
         items: replies.map(&:second),
-        next: last_id ? ActivityPub::TagManager.instance.replies_uri_for(object, page: true, min_id: last_id) : nil
+        next: last_id ? ActivityPub::TagManager.instance.replies_uri_for(object, page: true, min_id: last_id) : ActivityPub::TagManager.instance.replies_uri_for(object, page: true, other_accounts: true)
```

Comment:
I feel like `other_accounts` is an unhelpful param name. I have no idea what it does. Initially I assumed it would be an array of accounts!

Reply:
What should it be, then? When it isn't set, the endpoint will return self-replies only, when it is set, it will return only replies from other accounts.

Reply:
I think `only_self_replies` could be a good alternative

Reply:
`only_other_accounts` to not change the logic?

Reply:
Okay.